### PR TITLE
Add Logo Component

### PR DESCRIPTION
**Pull request by TheSolian**
Implemented the logo as a react component

### CODE REVIEW

**Review by h0peRL — DISMISSED** [review]
Everything work!
👍

File: usg-website/src/css/App.css [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 .App-logo {
     height: 40vmin;
-    pointer-events: none;
```
Comment (mnaray) [inline usg-website/src/css/App.css:7]: Very important to remove! Nice job.

File: usg-website/src/tests/App.test.tsx [hunk 2]
```diff
@@ -1,15 +1,16 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
 import App from '../pages/App';
 
 test('renders welcome text', () => {
-  render(<App />);
+  render(<App />, {wrapper: BrowserRouter});
```
Comment (mnaray) [inline usg-website/src/tests/App.test.tsx:7]: We need to remember to add the wrapper for other tests (if it has a link).

**Review by mnaray — APPROVED** [review]
Nice job. Thanks for the help.

**Pull request merged**